src/lib/services/client-metrics/metrics-service-v2.ts

```diff
@@ -62,23 +62,27 @@ export default class ClientMetricsServiceV2 {
         clientIp: string,
     ): Promise<void> {
         const value = await clientMetricsSchema.validateAsync(data);
-        const toggleNames = Object.keys(value.bucket.toggles);
+        const toggleNames = Object.keys(value.bucket.toggles).filter(
+            (name) =>
+                !(
+                    value.bucket.toggles[name].yes === 0 &&
+                    value.bucket.toggles[name].no === 0
+                ),
+        );
         if (toggleNames.length > 0) {
             await this.featureToggleStore.setLastSeen(toggleNames);
```

Comment:
this one has proven to be a bit costly to run on every metrics report as it end up doing an entire lock on the features table in the database. I will look in to moving this in to a separate service and optimize it a bit more.